src/lib/assets/hooks/collectibles.ts

```diff
@@ -15,218 +13,111 @@ import { getAccountAssetsStoreKey } from 'app/store/tezos/assets/utils';
-            slug,
-            status: getAssetStatus(balancesRecord[chainId]?.[slug], status),
-            chainId
-          }))
-        );
-      }
+      accountCollectibles = accountCollectibles.concat(
+        Object.entries(chainTokensRecord).map<AccountCollectible>(([slug, { status }]) => ({
+          slug,
+          status: getAssetStatus(balancesRecord[chainId]?.[slug], status),
+          chainId
+        }))
+      );
+    }
 
-      return accountCollectibles;
-    },
-    [enabledChains, collectiblesRecord, balancesRecord],
-    isEqual
-  );
+    return accountCollectibles.filter(c => c.status !== 'removed');
+  }, [enabledChains, collectiblesRecord, balancesRecord]);
 };
 
-const useEvmChainAccountCollectibles = (account: HexString, chainId: number) => {
+/**
+ * @returns Collectibes of account for chain, that were not removed
+ */
+export const useEvmChainAccountCollectibles = (account: HexString, chainId: number) => {
   const stored = useRawEvmChainAccountCollectiblesSelector(account, chainId);
   const balances = useRawEvmChainAccountBalancesSelector(account, chainId);
 
-  return useMemoWithCompare<AccountCollectible[]>(
-    () => {
-      const result: AccountCollectible[] = [];
-
-      for (const [slug, { status }] of Object.entries(stored)) {
-        if (status !== 'removed') result.push({ slug, status: getAssetStatus(balances[slug], status), chainId });
-      }
-
-      return result;
-    },
-    [stored, balances, chainId],
-    isEqual
-  );
-};
-
-export const useEnabledEvmAccountCollectiblesSlugs = (publicKeyHash: HexString) => {
-  const collectibles = useEvmAccountCollectibles(publicKeyHash);
-
-  return useMemo(
-    () =>
-      collectibles
-        .filter(({ status }) => status === 'enabled')
-        .map(({ chainId, slug }) => toChainAssetSlug(TempleChainKind.EVM, chainId, slug)),
-    [collectibles]
-  );
-};
-
-export const useAllEvmAccountCollectiblesSlugs = (publicKeyHash: HexString) => {
-  const collectibles = useEvmAccountCollectibles(publicKeyHash);
-
-  return useMemo(
-    () =>
-      collectibles
-        .filter(({ status }) => status !== 'removed')
-        .map(({ chainId, slug }) => toChainAssetSlug(TempleChainKind.EVM, chainId, slug)),
-    [collectibles]
-  );
-};
-
-export const useEnabledEvmChainAccountCollectiblesSlugs = (publicKeyHash: HexString, evmChainId: number) => {
-  const collectibles = useEvmChainAccountCollectibles(publicKeyHash, evmChainId);
-
-  return useMemo(
-    () => collectibles.filter(({ status }) => status === 'enabled').map(({ slug }) => slug),
-    [collectibles]
-  );
-};
+  return useMemoWithCompare<AccountCollectible[]>(() => {
+    const result: AccountCollectible[] = [];
 
-export const useAllEvmChainAccountCollectiblesSlugs = (publicKeyHash: HexString, evmChainId: number) => {
-  const collectibles = useEvmChainAccountCollectibles(publicKeyHash, evmChainId);
+    for (const [slug, { status }] of Object.entries(stored)) {
```

Comment:
Here is the same code as for the comment above, please consider additional refactoring